azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -265,7 +266,7 @@ public FlowRunnerManager(final Props props,
     this.cleanerThread = new CleanerThread();
     this.cleanerThread.start();
 
-    if (this.azkabanProps.getBoolean(ConfigurationKeys.AZKABAN_POLL_MODEL, false)) {
+    if (DispatchMethod.getDispatchModel(azkabanProps) == DispatchMethod.POLL) {
```

Comment:
What happens if the `azkaban.execution.dispatch.method` property isn't well configured and `DispatchMethod.valueOf` throws an IllegalArgumentException exception?

Reply:
Thanks. Took care of this comment.